mod-fqm-manager/src/main/resources/corsair/mod-fqm-manager/fqm-junit.feature

```diff
@@ -73,6 +73,7 @@ Feature: mod-fqm-manager integration tests
       | 'inventory-storage.instance-types.collection.get'           |
       | 'user-tenants.collection.get'                               |
       | 'inventory-storage.classification-types.collection.get'     |
+      | 'inventory-storage.instance-date-types.collection.get'      |
```

Comment:
We should probably add this to the mod-lists tests too, just to avoid any potential future problems.